src/io/flutter/editor/IndentsPass.java

```diff
@@ -180,12 +190,76 @@
     }
   };
 
+  // TODO(jacobr): some of this logic to compute what portion of the guide to
+  //  render is duplicated from RENDERER.
+  static io.flutter.editor.LineRange getGuideLineRange(Editor editor, RangeHighlighter highlighter) {
```

Comment:
Can you open a more specific issue about what we'd need (in our test infra) to be able to test this? Perhaps also w/ links to the tests that JetBrains has.

Reply:
Added some documentation to WidgetIndentsPass describing how I would want to test it.
As far as JetBrains tests go. I can't find any. In particular there aren't any references to the IntelliJ class outside of the code directory so any tests taht relate to this are integration tests.